gcp_variant_transforms/libs/vcf_reserved_fields.py

```diff
@@ -0,0 +1,95 @@
+# Copyright 2018 Google Inc.  All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Provides reserved INFO and FORMAT fields based on VCF 4.3 spec ."""
+
+from __future__ import absolute_import
+
+from vcf import parser
+
+_Format = parser._Format
+
+
+def _get_field_count(value):
+  return parser.field_counts[value]
+
+
+def _create_info(field, number, field_type, description):
```

Comment:
nit: please add `# type` notation here (particularly, b/c number is an int here; we are planning to change it to a string soon, so this needs to be slightly changed).